content/en/agent/guide/agent-status-page.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Agent Status Page
+kind: guide
+further_reading:
+- link: "agent/troubleshooting/"
+  tag: "Documentation"
+  text: "Agent Troubleshooting"
+- link: "agent/guide/agent-configuration-files/"
+  tag: "Guide"
+  text: "Agent Configuration Files"
+- link: "agent/guide/agent-commands/"
+  tag: "Guide"
+  text: "Agent Commands"
+---
+
+The Agent status page displays information about your running Agent. See [Agent Commands][1] to find the status command for your environment. The sections below provide details on the contents of the status page.
+
+## Agent version
+
+General information for Agent is displayed under the Agent version, for example:
+```text
+  Status date: 2019-08-29 18:16:41.526203 UTC
+  Agent start: 2019-08-29 18:04:18.060507 UTC
+  Pid: 12141
+  Go Version: go1.11.5
+  Python Version: 2.7.16
+  Check Runners: 4
+  Log Level: info
+```
+
+### Paths
+
+This section displays the paths to the Datadog config file, `conf.d` directory, and `checks.d` directory, for example:
+```text
+    Config File: /etc/datadog-agent/datadog.yaml
+    conf.d: /etc/datadog-agent/conf.d
+    checks.d: /etc/datadog-agent/checks.d
+```
+
+### Clocks
+
+This section displays the [NTP offset][2] and system UTC time, for example:
+```text
+    NTP offset: 2.095ms
+    System UTC time: 2019-08-29 18:16:41.526203 UTC
+```
+
+### Host Info
+
+This section displays information on the host the Agent is running on, for example:
+```text
+    bootTime: 2019-08-29 18:01:27.000000 UTC
+    kernelVersion: 4.4.0-109-generic
+    os: linux
+    platform: ubuntu
+    platformFamily: debian
+    platformVersion: 16.04
+    procs: 175
+    uptime: 2m53s
+    virtualizationRole: guest
+    virtualizationSystem: vbox
+```
+
+### Hostnames
+
+This section displays the hostnames found by the Agent, for example:
```

Comment:
We may want to explain that `hostname` is the final hostname reported to the backend where the other lines are explanation of what we found and may contain host aliases, no ?